deploy/operator/utils.sh

```diff
@@ -123,10 +123,10 @@ function wait_for_cmd_amount() {
     interval="$2"
     cmd="$3"
 
-    until [ $(${cmd} | wc -l) -eq ${amount} ]; do
+    until [ $(${cmd} "${@:4}" | wc -l) -eq ${amount} ]; do
```

Comment:
Would prefer to create a var in a similar fashion to 'cmd'

I.e 

arguments = "${@:4}"

Reply:
Since it is multiple arguments, it seems better this way

Reply:
Ok